.travis.yml

```
@@ -9,8 +9,6 @@ dist: bionic
 
 php:
   - 7.1
-  - 7.2
-  - 7.3
   - 7.4
```

```suggestion
  - 7.2
  - 7.3
  - 7.4
```

I can't push commits to this repo.